Gruntfile.js

```diff
@@ -63,16 +83,28 @@ module.exports = function(grunt) {
         watch: {
             js: {
                 files: ['js/**/*.js'],
-                tasks: ['concurrent', 'concat', 'uglify']
+                tasks: ['jslint', 'concat', 'uglify']
+            },
+            css: {
+                files: ['css/**/*.scss'],
+                tasks: ['scsslint', 'sass']
             }
         },
 
         concurrent: {
-            lint: ['jshint', 'jscs']
+            jslint: ['jslint'],
+            js: [['concurrent:jslint', 'concat', 'uglify']],
+            css: [['scsslint', 'sass']],
+            master: ['concurrent:js', 'concurrent:css']
```

Comment:
Hey Aurelio,

Locally I've updated the concurrent code to:

```

    concurrent: {
        js: [['jslint', 'concat', 'uglify']],
        css: [['scsslint', 'sass']]
    }

```

And when I register the tasks:

```

grunt.registerTask('default', ['concurrent']);
grunt.registerTask('dev', ['concurrent', 'watch']);

```

However, the reason I added the `master` task was to make sure the `js` and `css` ran at the same time.